mwdb/model/user.py

```diff
@@ -201,22 +201,52 @@ def generate_set_password_token(self):
             expiration=14 * 24 * 3600,
         )
 
+    def generate_group_invite_token(self, group_id, expiration):
+        return self._generate_token(
+            user_fields=["identity_ver"],
+            scope=AuthScope.group_invite,
+            expiration=expiration,
+            group_id=group_id,
+        )
+
     @staticmethod
-    def verify_session_token(token) -> Optional[Tuple["User", Optional[str]]]:
-        return User._verify_token(
+    def verify_group_invite_token(token: str) -> Optional[Tuple["User", str]]:
+        result = User._verify_token(
+            token,
+            ["identity_ver"],
+            scope=AuthScope.group_invite,
+        )
+        if result is None:
+            return None
+        else:
```

Comment:
`else` after `return` isn't necessary (unless used for performance reasons in an extremely often called function).